tools/proteore_clusterprofiler/cluster_profiler.xml

```diff
@@ -0,0 +1,314 @@
+                                <param name="ncol" type="text" value="c1" label="Column number of IDs" help='For example, fill in "c1" if it is the first column, "c2" if it is the second column and so on'>
+                                    <validator type="regex" message="Please enter a column number, for example: 'c1' for the first column">[c]{0,1}[0-9]+</validator>
+                                </param>
+                            </when>
+                        </conditional>
+                        <conditional name="universe_idti" >
+                            <param name="universe_idtypein" type="select" label="Select type of IDs of your background" help="" >
+                                <option value="Uniprot">UniProt Accession number</option>
+                                <option value="Entrez">Entrez Gene ID</option>
+                            </param>
+                            <when value="Uniprot"/>
+                            <when value="Entrez"/>
+                        </conditional>
+                    </when>
+                    <when value="false"/>
+                </conditional>
+                <param name="plot" type="select" display="checkboxes" multiple="true" label="Graphical display" optional="false">
+                    <option selected = "true" value="dotplot">dot-plot</option>
+                    <option value="barplot">bar-plot</option>
+                </param>
+            </when>
+            <when value="false"/>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="log" format="tsv" label="Cluster profiler" />
+        <collection type="list" label="clusterProfiler text files" name="text_output">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+\.tsv)" ext="tsv"/>
+        </collection>
+        <collection type="list" label="clusterProfiler diagram outputs" name="graph_output" >
+	        <discover_datasets pattern="(?P&lt;designation&gt;.+plot)" ext="png" />
+	    </collection>
+    </outputs>
+    <tests>
+        <test>
+            <conditional name="input">
+                <param name="ids" value="file"/>
+                <param name="file" value="Lacombe_et_al_2017_OK.txt"/>
+                <param name="header" value="true"/>
+                <param name="ncol" value="c1"/>
+            </conditional>
+            <conditional name="idti">
+                <param name="idtypein" value="Uniprot"/>
+            </conditional>
+            <param name="species" value="org.Hs.eg.db"/>
+            <conditional name="ggo">
+                <param name="go_represent" value="true"/>
+                <param name="level" value="3"/>
+            </conditional>
+            <conditional name="ego">
+                <param name="go_enrich" value="true"/>
+                 <conditional name="universe_input">
```

Comment:
Can you maybe add `count` to those tests, so we count the number of collection elements? https://docs.galaxyproject.org/en/latest/dev/schema.html#id96